### PR TITLE
Add addresses endpoints

### PR DESCRIPTION
**Pull request by alokg95**
/addresses endpoints + tests

### CODE REVIEW

File: dispatchr-ruby/spec/requests/addresses_spec.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe "Addresses", :type => :request do
+  describe "GET /addresses" do
+    it "works! (now write some real specs)" do
```
Comment (salolivares) [inline dispatchr-ruby/spec/requests/addresses_spec.rb]: Is this valid ruby? Sorry I don't know how rspec really works.

File: dispatchr-ruby/app/models/address.rb [hunk 2]
```diff
@@ -1,4 +1,8 @@
 class Address < ApplicationRecord
   has_many :users
   has_many :stores
+
+  validates :address, presence: true
```
Comment (salolivares) [inline dispatchr-ruby/app/models/address.rb]: Mind explaining what these lines do?

File: dispatchr-ruby/spec/requests/addresses_spec.rb (hunk 1 above)
Comment (JordanNguyen) [inline dispatchr-ruby/spec/requests/addresses_spec.rb]: this is auto generated from scaffolding, dont worry about it :)

File: dispatchr-ruby/spec/controllers/addresses_controller_spec.rb [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+        expect(parsed_response['longitude']).to eq('2.0')
+        expect(parsed_response['latitude']).to eq('2.0')
+      end
+    end
+  end
+
+  describe 'DELETE #destroy' do
+    context 'when id does not exist' do
+      it 'raises error' do
+        expect {put '/addresses/-1'}.to raise_error
```
Comment (JordanNguyen) [inline dispatchr-ruby/spec/controllers/addresses_controller_spec.rb]: typo. put -> delete

**Comment by JordanNguyen** [comment]
looks fine except for 1 typo, which doesnt really make a difference. fix that and lets merge

**Pull request merged**